Index: team_one_project/src/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import * as React from \"react\";\n// import { createRoot } from \"react-dom/client\";\n// import {\n//   createBrowserRouter,\n//   RouterProvider,\n//   Route,\n//   Link,\n// } from \"react-router-dom\";\nimport Links from \"./components/Links\";\nimport MainSideBar from \"./components/MainSideBar\";\n\nfunction App() {\n  return (\n    <>\n      <Links />\n      <MainSideBar />\n    </>\n  );\n}\n\nexport default App;\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/team_one_project/src/App.js b/team_one_project/src/App.js
--- a/team_one_project/src/App.js	(revision 31c6db3c6b4e1a70ede50b9e9805f3ab5be95253)
+++ b/team_one_project/src/App.js	(date 1695705665481)
@@ -8,12 +8,14 @@
 // } from "react-router-dom";
 import Links from "./components/Links";
 import MainSideBar from "./components/MainSideBar";
+import LoginSignup from "./components/login.rg.fpw/LoginSignup";
 
 function App() {
   return (
     <>
       <Links />
       <MainSideBar />
+      <LoginSignup/>
     </>
   );
 }
Index: package-lock.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/package-lock.json b/package-lock.json
new file mode 100644
--- /dev/null	(date 1695610133558)
+++ b/package-lock.json	(date 1695610133558)
@@ -0,0 +1,6 @@
+{
+  "name": "CS_370_Team_One",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
